engine/execution/computation/manager.go

```diff
@@ -170,42 +213,15 @@ func (e *Manager) ExecuteScript(
 	blockCtx := fvm.NewContextFromParent(e.vmCtx, fvm.WithBlockHeader(blockHeader))
 	programs := e.getChildProgramsOrEmpty(blockHeader.ID())
 
-	err := func() (err error) {
-
-		start := time.Now()
-
-		defer func() {
-
-			prepareLog := func() *zerolog.Event {
-
-				args := make([]string, 0)
-				for _, a := range arguments {
-					args = append(args, hex.EncodeToString(a))
-				}
-				return e.log.Error().
-					Hex("script_hex", code).
-					Str("args", strings.Join(args[:], ","))
-			}
-
-			elapsed := time.Since(start)
-
-			if r := recover(); r != nil {
-				prepareLog().
-					Interface("recovered", r).
-					Msg("script execution caused runtime panic")
-
-				err = fmt.Errorf("cadence runtime error: %s", r)
```

Comment:
The bug is that setting the `err` variable on this line doesn't actually cause the error to be returned, because the parent function `ExecuteScript` does not declare `err` as a named return variable